src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { memo } from 'react';
+import cn from 'classnames';
+
+type Props = {
+  isError: boolean;
+  onCloseErrorMessage: () => void;
+  errorMessage: string;
+};
+
+export const ErrorNotification: React.FC<Props> = memo((props) => {
+  const {
+    isError,
+    onCloseErrorMessage,
+    errorMessage,
+  } = props;
+
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className={cn(
+        'notification',
+        'is-danger',
+        'is-light',
+        'has-text-weight-normal',
+        { hidden: !isError },
+      )}
+    >
+      {/* eslint-disable jsx-a11y/control-has-associated-label */}
+      <button
+        data-cy="HideErrorButton"
+        type="button"
+        className="delete"
+        onClick={onCloseErrorMessage}
```

Comment:
Imagine function names like you ask them to do something for you (ask for some action)
Fix here and in all other places

For example `onClick` ask to `closeErrorMessage`
```suggestion
        onClick={closeErrorMessage}
```